kong/plugins/response-ratelimiting/schema.lua

```diff
@@ -75,6 +75,9 @@ return {
           { redis_port = typedefs.port({ default = 6379 }), },
           { redis_password = { type = "string", len_min = 0, referenceable = true }, },
           { redis_username = { type = "string", referenceable = true }, },
+          { redis_ssl = { type = "boolean", required = true, default = false, }, },
+          { redis_ssl_verify = { type = "boolean", required = true, default = false }, },
+          { redis_server_name = typedefs.sni },
```

Comment:
> I don't think this is a breaking change.

@ADD-SP Just a heads up, this is a breaking change for version compatibility with older data planes; updates will need to be made to the [compatibility layer ](https://github.com/Kong/kong/blob/master/kong/clustering/compat/removed_fields.lua)to handle these changes.

@aboudreault version compatibility issues to watch out for.

Reply:
@dominikkukacka Thanks for the offer, but right now with the version compatibility tests being in a non-perfect state and this feature is currently being targeted for a particular release (going in `master` for now) this is something that can be added closer to release time. I just wanted to identify this as being something that needed to be handled before the release.  Sorry for the confusion.

Reply:
There is no official date right now for the next release.